Compilation errors on master

I'm seeing two errors in building master from source.

Running `cargo build --all --release --bins --lib` now generates the following error.

```
warning: unused import: `siginfo_ext::SiginfoExt`
 --> lucet-runtime/lucet-runtime-internals/src/instance/signals.rs:3:5
  |
3 |     siginfo_ext::SiginfoExt, FaultDetails, Instance, State, TerminationDetails, CURRENT_INSTANCE,
  |     ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default

warning: unused import: `crate::instance::siginfo_ext::SiginfoExt`
  --> lucet-runtime/lucet-runtime-internals/src/instance.rs:10:5
   |
10 | use crate::instance::siginfo_ext::SiginfoExt;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0133]: call to unsafe function is unsafe and requires unsafe function or block
   --> lucet-runtime/lucet-runtime-internals/src/instance.rs:842:25
    |
842 |                         siginfo.si_addr()
    |                         ^^^^^^^^^^^^^^^^^ call to unsafe function
    |
    = note: consult the function's documentation for information on how to avoid undefined behavior

error[E0133]: call to unsafe function is unsafe and requires unsafe function or block
   --> lucet-runtime/lucet-runtime-internals/src/instance/signals.rs:192:55
    |
192 |                     && !inst.alloc.addr_in_heap_guard(siginfo.si_addr());
    |                                                       ^^^^^^^^^^^^^^^^^ call to unsafe function
    |
    = note: consult the function's documentation for information on how to avoid undefined behavior

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0133`.
error: Could not compile `lucet-runtime-internals`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```
The second issue is stranger. When I run `make build` it launches minimalist Go framework on port 8080 and just sits there. Very strange.
```

   ____    __
  / __/___/ /  ___
 / _// __/ _ \/ _ \
/___/\__/_//_/\___/ v4.1.5
High performance, minimalist Go web framework
https://echo.labstack.com
____________________________________O/_______
                                    O\
⇨ http server started on [::]:8080
```

--- a/lucet-runtime/lucet-runtime-internals/src/instance.rs
+++ b/lucet-runtime/lucet-runtime-internals/src/instance.rs
@@ -839,7 +839,7 @@ impl std::fmt::Display for State {
                     write!(
                         f,
                         " accessed memory at {:p} (inside heap guard)",
-                        siginfo.si_addr()
+                        siginfo.si_addr_ext()
                     )?;
                 }
                 Ok(())

--- a/lucet-runtime/lucet-runtime-internals/src/instance/siginfo_ext.rs
+++ b/lucet-runtime/lucet-runtime-internals/src/instance/siginfo_ext.rs
@@ -5,11 +5,11 @@ extern "C" {
 }
 
 pub trait SiginfoExt {
-    fn si_addr(&self) -> *const c_void;
+    fn si_addr_ext(&self) -> *const c_void;
 }
 
 impl SiginfoExt for siginfo_t {
-    fn si_addr(&self) -> *const c_void {
+    fn si_addr_ext(&self) -> *const c_void {
         unsafe { siginfo_si_addr(self as *const siginfo_t) }
     }
 }

--- a/lucet-runtime/lucet-runtime-internals/src/instance/signals.rs
+++ b/lucet-runtime/lucet-runtime-internals/src/instance/signals.rs
@@ -189,7 +189,7 @@ extern "C" fn handle_signal(signum: c_int, siginfo_ptr: *mut siginfo_t, ucontext
                 // If the trap was a segv or bus fault and the addressed memory was outside the
                 // guard pages, it is also a fatal error
                 let outside_guard = (siginfo.si_signo == SIGSEGV || siginfo.si_signo == SIGBUS)
-                    && !inst.alloc.addr_in_heap_guard(siginfo.si_addr());
+                    && !inst.alloc.addr_in_heap_guard(siginfo.si_addr_ext());
 
                 // record the fault and jump back to the host context
                 inst.state = State::Fault {

